earthpy/clip.py

```diff
@@ -43,8 +44,31 @@ def clip_line_poly(shp, clip_obj):
 
 
 def clip_shp(shp, clip_obj):
-    '''
-    '''
+   """A function to clip points, lines, polygon geometries based on an input
+   geometry.
+
+   Both layers must be the same Coordinate Reference System (CRS).
+
+   Depending on the geometry type, input data will be clipped to the full
+   extent of clip_obj using either clip_points or clip_line_poly.
+
+   If there are multiple polygons in object,
+   data will be clipped to the total boundary of
+   all polygons.
+
+   Parameters
+   ----------
+   shp : Geopandas dataframe
+         Dataset to be clipped.
```

Comment:
Please update line 62 to `Vector layer (point, line, polygon) to be clipped to clip_obj`.